spec/models/user_spec.rb

```diff
@@ -17,5 +17,14 @@
     let(:user) { User.create_by_admin(email: 'lola@o2.pl')}
     it { user.valid_password?('secret').eql? true }
   end
+
+  describe '#to_csv' do
```

Comment:
But in fact ```to_csv``` is just calling ```UsersCsvExport``` and result from ```UsersCsvExport``` is already tested by specs for service. 

So the only thing that should be tested is - is ```UserCsvExport``` called with correct arguments?

https://relishapp.com/rspec/rspec-mocks/docs/basics/expecting-messages